synapse/app/generic_worker.py

```diff
@@ -237,6 +238,7 @@ class GenericWorkerSlavedStore(
     SlavedEventStore,
     SlavedKeyStore,
     RoomWorkerStore,
+    RoomBatchStore,
```

Comment:
@tulir Might be good to split out this missing import fix for worker mode in a separate PR so we can merge it while we figure out the other problem.